cobrawap/pipeline/stage02_processing/Snakefile

```diff
@@ -23,14 +23,14 @@ use rule template_all as all with:
     input:
         check = OUTPUT_DIR / 'input.check',
         data = input_file,
-        img = OUTPUT_DIR / f'processed_traces_{config.PLOT_TSTART}-{config.PLOT_TSTOP}s',
+        img = OUTPUT_DIR / str("processed_traces_%s-%ss" % (config.PLOT_TSTART,config.PLOT_TSTOP)),
         # configfile = Path('configs') / f'config_{PROFILE}.yaml'
 
 rule clear:
     output:
         temp(Path('{path}') / 'clear.done')
     params:
-        block_folder = [Path('{path}') / f'{block}' for block in config.BLOCK_ORDER]
+        block_folder = [Path('{path}') / str('{block}') for block in config.BLOCK_ORDER]
```

Comment:
```suggestion
        block_folder = [Path('{path}') / block for block in config.BLOCK_ORDER]
```
this interprets '{block}' as literal, causing an error because ../path/'{block}' doesn't exist.